### ISSUE TITLE
Feature request - Bulk "Cancel Submissions"

### ISSUE THREAD

**Issue by johanneskruse** [codalab/codabench#1487]
Hi, 

Thank you for the awesome platform.

I'm running a competition, and due to some technical issues, I've had to cancel and resubmit a lot of submissions.

Hence, a feature suggestion would be to add a "Cancel Submissions" option similar to "Rerun/Delete Submissions".

![image](https://github.com/codalab/codabench/assets/43001334/518f3ae2-30c4-4b2f-835e-4b68858e1d50)

Best,
Johannes

**Comment by Didayolo** [comment]
Interesting suggestion. However, as long as the cancellation does not really prevents computation (#872), this is not that relevant.

**Comment by johanneskruse** [comment]
@Didayolo, so the cancellation doesn't really do anything?

**Comment by Didayolo** [comment]
@johanneskruse Actually it depends. If the submission is still in "Submitted" status, still not sent to a worker, then it is cancelled for real (from what I know). However, if it is already running, cancelling it won't stop the on going computation from the compute worker.

**Comment by johanneskruse** [comment]
Makes sense, thanks for the clarification. During the issues encountered with #1473 I usually cancel submissions when they get congested (and backlog becomes too large, and people started submitted the same submissions over and over again), also to make it visible for participants that it is no longer submitting.
